Capri.Web.Services/ServiceResult.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Capri.Web.ViewModels;
+
+namespace Capri.Services
+{
+    public class ServiceResult<T>
```

Comment:
That ServiceResult is far from what I've shown you how to do it. Acceptance criteria for it:

- inherit from Dictionary<string, string>
- contain IsSucceded property (getter)
- contain methods like:
`        
void AddError(string key, string errorMessage);

void AddError(string key, Exception exception);
`

- contain object of class which represents in TBody property (getter)
- contain static methods like ServiceResult<T>.Success<T>(T body), ServiceResult.Error(params string[] errors)